tableview/build.gradle

```diff
@@ -47,11 +46,55 @@ dependencies {
     implementation "androidx.recyclerview:recyclerview:$androidx_recyclerview_version"
 }
 
-publish {
-    userOrg = 'evrencoskun' //user name of bintray.com
-    groupId = 'com.evrencoskun.library' //jcenter's url
-    artifactId = 'tableview' //project name
-    publishVersion = '0.8.9' //version code
-    desc = 'TableView is a powerful Android library for displaying complex data structures and rendering tabular data composed of rows, columns and cells.'
-    website = 'https://github.com/evrencoskun/TableView' //website
+// Configure the publishing
+apply plugin: 'maven-publish'
```

Comment:
Since Android Gradle Plugin 3.6, Gradle's Maven Publish Plugin is supported for Android projects.
I configured it so you don't depend on third party plugins. If you prefer to use an existing plugin, let me know and I'll revert my changes.
I can also revert to the latest Gradle 5 release, if you want to stay with `com.novoda.bintray-release` plugin

I tried this change on my side by running `./gradlew :tableview:publishReleasePublicationToMavenLocal, and I ended up with the AAR file, the corresponding POM file with the dependencies, the sources JAR and the Javadoc JAR. I couldn't test a publication on the actual Bintray repository though.